mlem/core/objects.py

```diff
@@ -866,25 +1225,33 @@ def wait_for_status(
                 return False
             time.sleep(timeout)
         if raise_on_timeout:
+            # TODO: count actual time passed
```

Comment:
Do you plan to address it in a separate PR? Let's create an issue then?

Reply:
https://github.com/iterative/mlem/issues/403